pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -364,6 +365,15 @@ func (r *ReconcileComplianceScan) phaseDoneHandler(instance *compv1alpha1.Compli
 			log.Error(err, "Cannot delete CA secret")
 			return reconcile.Result{}, err
 		}
+
+		if instance.NeedsRescan() {
+			instanceCopy := instance.DeepCopy()
+			instanceCopy.Status.Phase = compv1alpha1.PhasePending
+			instanceCopy.Status.CurrentIndex = instance.Status.CurrentIndex + 1
```

Comment:
Do you know offhand how does golang handle integer overflow? I guess we won't go over INT_MAX anytime soon, but the C programmer in me doesn't like increasing an int without limits anyway :-)
(in C, the typical way to solve this might be if increased < previous { increased = 0 } or similar)

Reply:
Good question, gotta research it.

Reply:
Addressed